addons/hr/models/hr_employee.py

```diff
@@ -1217,7 +1242,7 @@ def _get_calendars(self, date_from=None):
 
         date_from = fields.Date.to_date(date_from)
         for employee in self:
-            employee_versions = employee.version_ids.filtered(lambda v: v._is_in_contract(date_from))
+            employee_versions = employee.version_ids.sudo().filtered(lambda v: v._is_in_contract(date_from))
```

Comment:
nitpicking just so we know :innocent: 
```suggestion
            employee_versions_sudo = employee.version_ids.sudo().filtered(lambda v: v._is_in_contract(date_from))
```